app/views/layouts/_topbar.html.haml

```diff
@@ -2,8 +2,8 @@
   %ul.title-area
     %li.name
 
-      %h1#shostakodech
-        %a#home{:href => root_path} Catechumen
+      %h1#catechumen
+        %a#home{href: root_path} Catechumen
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace